draft-ietf-tls-tls13.md

```diff
@@ -3641,9 +3642,11 @@ single connection, either immediately after each other or
 after specific events.
 For instance, the server might send a new ticket after post-handshake
 authentication in order to encapsulate the additional client
-authentication state. Clients SHOULD attempt to use each
-ticket no more than once, with more recent tickets being used
-first.
+authentication state. Multiple tickets are useful for clients that 
+perform connection racing across interfaces and address families 
+via, e.g., Happy Eyeballs {{RFC6555}} or related techniques.
```

Comment:
If were going to add text, I think you should also  mention that you might do multiple HTTP connections

Reply:
Done.